packages/gitmoji-changelog-core/src/utils.js

```diff
@@ -0,0 +1,32 @@
+const levenshtein = require('fast-levenshtein')
+
+function groupSentencesByDistance(texts = []) {
+  const textsWithSortedWords = texts.map(text => text.split(' ').sort().join(''))
+
+  const alreadyProcessedWords = new Set()
+  const keyGroups = new Map()
+
+  for (let i = 0; i < textsWithSortedWords.length; i += 1) {
```

Comment:
Why i += 1 instead of i++ ?

Reply:
Because people are not familiar about this difference between ++i and i++ I guess ?
This is a linter rule I have in other projects, I don't try to trigger it in gitmoji-changelog

Reply:
Here is a the real reason: https://eslint.org/docs/rules/no-plusplus

This is about ++i or i++ but not for what I thank :)